compute/admin_guide/continuous_integration/code_repo_scanning.adoc

```diff
@@ -9,6 +9,7 @@ The runtimes supported are:
 * Node.js
 * Python
 * Ruby
+* C# (.NET Core 2.1, .NET Core 3.1) 
```

Comment:
@tallieber , please confirm. 

Related but not the same issue.
This is also contradictory to what we see specifically for serverless runtimes in the UI per https://spring.paloaltonetworks.com/twistlock/twistlock/issues/33209
vs
the runtimes displayed in the Open API Spec file for downloading serverless bundles: https://prisma.pan.dev/api/cloud/cwpp/defenders#operation/post-defenders-serverless-bundle
![image](https://user-images.githubusercontent.com/22352459/175122391-2bea1489-15cf-473f-ac62-e8ab24986954.png)

Would be great to get the correct list for what's supported.

Reply:
cc: @rcaballeromx

Reply:
Thanks all, if we support  .Net Framework (4.x) based libraries/executables, and .Net Core (5.x, 6.0) based libraries/executables should we add those as well?

Reply:
Hi @ipalvr , I believe @avish9 had told me that `only 3.1 and 6.0 are supported (4.0/5.0 arent supported).` It was in a review comment here: https://docs.google.com/document/d/1AhlagzchynFUOUe3D-v_zBiwiQ84oQ1z/edit

Please confirm

Reply:
The versions I mentioned relate to Runtime Protection - Serverless Defender for Azure.